### PR TITLE
release guide

### PR DESCRIPTION
**Pull request by dmsolr**

### CODE REVIEW

File: release.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+Mail content:
+Hi all,
+
+Apache SkyWalking  Team is glad to announce the first release of Apache SkyWalking Kong x.y.z
+
+SkyWalking: APM (application performance monitor) tool for distributed systems,
+especially designed for microservices, cloud native and container-based (Docker, Kubernetes, Mesos) architectures.
+
+SkyWalking Nginx Agent provides the native tracing capability for Nginx powered by Kong module.
```
Comment (wu-sheng) [inline release.md]: Copy/paste issue.

File: release.md [hunk 2]
```diff
@@ -109,13 +109,11 @@ Mail title: [ANNOUNCE] Release Apache SkyWalking Kong version x.y.z
 Mail content:
 Hi all,
 
-Apache SkyWalking  Team is glad to announce the first release of Apache SkyWalking Kong x.y.z
+Apache SkyWalking Team is glad to announce to release Apache SkyWalking Kong version x.y.z.
 
 SkyWalking: APM (application performance monitor) tool for distributed systems,
 especially designed for microservices, cloud native and container-based (Docker, Kubernetes, Mesos) architectures.
 
-SkyWalking Nginx Agent provides the native tracing capability for Nginx powered by Kong module.
```
Comment (wu-sheng) [inline release.md]: You should keep this

> SkyWalking Kong Agent provides the native tracing capability for Kong gateway.

**Review by wu-sheng — APPROVED** [review]

File: release.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+
+Voting will start now (xxxx date) and will remain open for at least 72 hours, Request all PMC members to give their vote.
+[ ] +1 Release this package.
+[ ] +0 No opinion.
+[ ] -1 Do not release this package because....
+
+```
+
+5. Publish release, if vote passed.
```
Comment (kezhenxu94) [inline release.md:94]: Better to add a step to remove old unrecommended releases in the svn

Comment (dmsolr) [reply release.md:94]: I raise a new PR soon.

**Pull request merged**